src/components/BookState/BookState.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './BookState.scss';
+
+function BookState() {
+  return (
+    <div className="book-container">
+      <div className="book-info-container">
+        <h2 className="book-genre">
+          Action
+        </h2>
+        <h3 className="book-title">
+          The Hunger Games
+        </h3>
+        <h4 className="book-author">
+          Suzanne Collins
+        </h4>
+      </div>
+      <ul className="options-container">
+        <li><a href="/">Comments</a></li>
+        <li><a href="/">Remove</a></li>
+        <li><a href="/">Edit</a></li>
+      </ul>
+    </div>
+  );
+}
```

Comment:
- [ ] Based on the project requirement, I suggest this component `BokkState` should be split into two components; 
   - `Book` component which contains the HTML markup for the individual book. 
       - It should take the book data as a prop and renders the title and the author
      * Has a button for removing books
      * The button wouldn't remove the books *yet*, this is just a placeholder for now and should receive `props`
   - `BookList` should render a list of books using the `<Book>` component via a higher order function `map`. that means that the book data should be an array. 


Kindly implement this as it is a core requirement. 🙏🏾 

Something like so 👇🏾 

```javascript
// Book.js
function Book({ book }) {
  return (
    <div className="book">
      <h2>{book.title}</h2>
      <p>{book.author}</p>
      <button>Remove</button>
     {
     // Other buttons
     }
    </div>
  );
}

// BookList.js
function BookList({ books }) {
  return (
    <div className="book-list">
      {books.map((book) => (
        <Book key={book.id} book={book} />
      ))}
    </div>
  );
}
```